### PR TITLE
refactor(client/server): lib zones/ox target/locales

### PR DESCRIPTION
**Pull request by Randolio**
## Description

#] Use ox lib zones, remove polyzone.
#] Switch qb-target exports to ox target.
#] Convert all locales into lib locale json format.
#] Support new vehicle finance sql
#] Some typo fixes and minor rewrites.

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

File: config/client.lua [hunk 1]
```diff
@@ -0,0 +1,43 @@
+return {
+    UseTarget = GetConvar('UseTarget', 'false') == 'true',
```
Comment (mafewtm) [inline config/client.lua:2]: We don't have a convar for this so this can just be defaulted to false. Also, utilize camelCase for all of these

**Review by mafewtm — APPROVED** [review]

File: config/client.lua (hunk 1 above)
Comment (Randolio) [reply config/client.lua:2]: ahh my bad, I was just following format of other resources but I guess they're not updated.

File: server/main.lua [hunk 2]
```diff
@@ -36,18 +36,30 @@ lib.callback.register('qbx_vehiclesales:server:spawnVehicle', function (source,
     return netId
 end)
 
+lib.callback.register('qbx_vehiclesales:server:checkVehicleOwner', function(source, plate)
+    local player = exports.qbx_core:GetPlayer(source)
+    local result = MySQL.single.await('SELECT * FROM player_vehicles WHERE plate = ? AND citizenid = ?', {plate, player.PlayerData.citizenid})
```
Comment (Manason) [inline server/main.lua:41]: Should call qbx_vehicles API instead of direct DB call here. Fine to leave for a future PR.

File: config/client.lua [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            businessName = "Vehicle Sales Contract - Larry's Vehicle Sales",
+            sellVehicle = vec4(1235.61, 2733.44, 37.4, 0.42),
+            buyVehicle = vec4(1213.31, 2735.4, 38.27, 182.5),
+            points = {
+                vec3(1338.3748779297, 2645.0153808594, 36.0),
+                vec3(1098.9381103516, 2621.7487792969, 36.0),
+                vec3(1117.9478759766, 2822.0729980469, 36.0),
+                vec3(1370.98828125, 2859.197265625, 36.0)
+            },
+            spots = {
```
Comment (Manason) [inline config/client.lua:14]: these names are unclear. What are points? spots? These just sound like synonyms of coordinates. Consider renaming to be more descriptive and/or add comments to describe what these fields mean

**Review by Manason — APPROVED** [review]

File: config/client.lua (hunk 3 above)
Comment (Randolio) [reply config/client.lua:14]: Well points would be the collection of points that form the zone. Spots are just the vehicle spots in which the for sale cars get placed. I can rename I guess.

**Review by TonybynMp4 — APPROVED** [review]

File: server/main.lua (hunk 2 above)
Comment (TonybynMp4) [reply server/main.lua:41]: Weird 🤔

Comment (Randolio) [reply server/main.lua:41]: Oh well, I'll leave it for a future pr.

**Pull request merged**